parlai/core/torch_agent.py

```diff
@@ -215,6 +215,13 @@ def reset(self):
         self.history_strings = []
         self.history_vecs = []
 
+    def reset_metrics(self):
```

Comment:
why is this in the History object?????? super().reset_metrics() here isn't going to do anything?